packages/google-maps-routing/package.json

```diff
@@ -57,6 +58,7 @@
     "jsdoc-fresh": "^2.0.0",
     "jsdoc-region-tag": "^2.0.0",
     "linkinator": "^4.0.0",
+    "long": "^5.2.3",
```

Comment:
I don't think we should need to add this `long` dependency, as it's pulled in by google-gax. I'm curious why this was needed.

Reply:
The way pnpm installs dependencies doesn't grab this dependency. It's really only for [this line](https://github.com/googleapis/google-cloud-node/blob/8db807ae80a695bb1ad4fa290bc9a50ce29a8365/packages/google-api-servicemanagement/protos/protos.d.ts#L16). I spoke with @alexander-fenster about this, and he recommended this option, over skipping type checking in the tsconfig.